flickit-advice-engine/src/main/java/org/flickit/assessment/advice/application/port/in/adviceitem/CreateAdviceItemUseCase.java

```diff
@@ -0,0 +1,70 @@
+package org.flickit.assessment.advice.application.port.in.adviceitem;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import org.flickit.assessment.advice.application.domain.adviceitem.CostLevel;
+import org.flickit.assessment.advice.application.domain.adviceitem.ImpactLevel;
+import org.flickit.assessment.advice.application.domain.adviceitem.PriorityLevel;
+import org.flickit.assessment.common.application.SelfValidating;
+import org.flickit.assessment.common.validation.EnumValue;
+
+import java.util.UUID;
+
+import static org.flickit.assessment.common.error.ErrorMessageKey.COMMON_CURRENT_USER_ID_NOT_NULL;
+import static org.flickit.assessment.advice.common.ErrorMessageKey.*;
+
+
+public interface CreateAdviceItemUseCase {
+
+    Result createAdviceItem(Param param);
+
+    @Value
+    @EqualsAndHashCode(callSuper = false)
+    class Param extends SelfValidating<Param> {
+
+        @NotNull(message = CREATE_ADVICE_ITEM_ASSESSMENT_ID_NOT_NULL)
+        UUID assessmentId;
+
+        @NotNull(message = CREATE_ADVICE_ITEM_TITLE_NOT_NULL)
+        @Size(min = 3, message = CREATE_ADVICE_ITEM_TITLE_SIZE_MIN)
+        @Size(max = 100, message = CREATE_ADVICE_ITEM_TITLE_SIZE_MAX)
+        String title;
+
+        @Size(min = 3, message = CREATE_ADVICE_ITEM_DESCRIPTION_SIZE_MIN)
+        @Size(max = 1000, message = CREATE_ADVICE_ITEM_DESCRIPTION_SIZE_MAX)
```

Comment:
It's been decided that the description field won't be nullable and will have a max length of 3,000 characters.